packages/curve-ui-kit/src/widgets/Footer/Sections.tsx

```diff
@@ -0,0 +1,120 @@
+import { t } from '@lingui/macro'
+
+import YouTubeIcon from '@mui/icons-material/YouTube'
+import TelegramIcon from '@mui/icons-material/Telegram'
+import NewspaperIcon from '@mui/icons-material/Newspaper'
+import IntegrationInstructionsOutlinedIcon from '@mui/icons-material/IntegrationInstructionsOutlined'
+import BookIcon from '@mui/icons-material/Book'
+import GitHubIcon from '@mui/icons-material/GitHub'
+import BeenhereOutlinedIcon from '@mui/icons-material/BeenhereOutlined'
+import BugReportOutlinedIcon from '@mui/icons-material/BugReportOutlined'
+
+import { XIcon } from 'curve-ui-kit/src/shared/icons/XIcon'
+import { DiscordIcon } from 'curve-ui-kit/src/shared/icons/DiscordIcon'
+import { LAFIcon } from 'curve-ui-kit/src/shared/icons/LAFIcon'
+import { DuneIcon } from 'curve-ui-kit/src/shared/icons/DuneIcon'
+import { CrvHubIcon } from 'curve-ui-kit/src/shared/icons/CrvHubIcon'
+import { SectionProps } from './Section'
+
+export const sections: SectionProps[] = [
```

Comment:
Calling `t` on the module level can give [issues when the user changes their language](https://lingui.dev/ref/macro#using-macros). However, just easily finding `t` references is good enough for now.

Reply:
I figured that'd be the case and thought it was good enough as well for the `t` reference finding. But then if we do decide to keeps translations I'm sure we'd forget about this, so I tried to fix it in https://github.com/curvefi/curve-frontend/pull/500/commits/8ba7f676a402493526e93a25e7d3f73be4a7d009. Wasn't able to test it, but when locale changes the footer should re-render and `getSection()` should be execugted again I think.